src/styles/blocks/catalog.scss

```diff
@@ -7,19 +7,25 @@
   padding: 50px 0;
 
   &__card {
-    display: flex;
+    @extend %display-element-properties;
     justify-content: center;
-    align-items: center;
-    box-sizing: border-box;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $animation-duration;
+    transition-duration: $animation-duration;
```

Comment:
You already have transition-duration rule in transition rule above

Reply:
Ooops. Something went wrong %)

Reply:
done